File: src/main/frontend/src/app/employees/employee.ts
import {DatePipe} from "@angular/common";

export class Employee {

    city: string;
    dob: string;
    emailAddress: string;
    empId: any;
    fname: string;
    lname: string;
    phoneNumber: string;
    state:string;
    streetAddress: string;
    zipCode: string;




    constructor(empId: any, fname: string, lname: string, streetAddress: string,
                city: string, state: string, zipCode: string, emailAddress: string, phoneNumber: string, dob:string){
        this.empId = empId;
        this.fname = fname;
        this.lname = lname;
        this.streetAddress = streetAddress;
        this.city = city;
        this.state = state;
        this.zipCode = zipCode;
        this.emailAddress = emailAddress;
        this.phoneNumber = phoneNumber;
        this.dob = dob;
    }


}

File: src/main/java/com/genuinelygreen/management/domain/Employee.java
package com.genuinelygreen.management.domain;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import java.time.LocalDate;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Employee {

    @Id
    @GeneratedValue
    private Long empId;

    private String fName;
    private String lName;
    private String streetAddress;
    private String city;
    private String state;
    private String zipCode;
    private String emailAddress;
    private String phoneNumber;
    @JsonFormat(pattern = "yyyy-MM-dd")
    private LocalDate dob;
}
